src/osd/OSDMap.h

```diff
@@ -36,9 +36,12 @@
 #include "include/memory.h"
 using namespace std;
 
-//forward declaration
+// forward declaration
 class CephContext;
 class CrushWrapper;
+class OSDMap;
```

Comment:
this is not necessary any more, as `OSDMapMapping` is declared in its own header file.